src/tiny-invariant.ts

```diff
@@ -1,9 +1,20 @@
 const isProduction: boolean = process.env.NODE_ENV === 'production';
 const prefix: string = 'Invariant failed';
 
-// Throw an error if the condition fails
-// Strip out error messages for production
-// > Not providing an inline default argument for message as the result is smaller
+/**
+ * Throw an error if the condition fails. Strip out error messages for production.
+ *
+ * @param condition A boolean condition - if falsey will thrown an error.
```

Comment:
Do we have to provide definitions for the `@param`s? It feels like doubling up 🤔

Reply:
I wonder if this is enough as a `jsdoc`:

```ts
/** Throw an `Error` if the `condition` is falsey */
```

Reply:
What do you think @DarkPurple141 ?

Reply:
Don't need them necessarily, really just there for completeness.